README.md

```diff
@@ -561,6 +561,58 @@ Use it to type in your key and install.
 
 Go back to Step 4A-4 and then finish with Step 4A-5.
 
+---
+## Forks
+
+### Which Fork Should I Use?
+
+> [!WARNING]
+> If you have to ask, you're not ready for forks.
```

Comment:
```suggestion
If you have to ask, you're not ready for forks.
```
Completely up to you on whether to take this change. With the caution box below, this seemed a bit redundant. Though on the other hand, plastering warning signs near a danger is also not a bad idea.

Reply:
I'm fine with this. I agree